voxblox/include/voxblox/integrator/merge_integration.h

```diff
@@ -221,7 +222,7 @@ void transformLayer(const Layer<VoxelType>& layer_in,
         layer_out->allocateBlockPtrByIndex(block_idx);
 
     for (IndexElement voxel_idx = 0;
-          voxel_idx < static_cast<IndexElement>(block->num_voxels());
+         voxel_idx < static_cast<IndexElement>(block->num_voxels());
```

Comment:
this was already fixe by @cwitting  but when I resolved a merge conflict in this file in another PR it must have undone his changes.